Cargo.toml

```
@@ -49,7 +50,7 @@ default-features = false
 # optional = true
 
 [dependencies.rand_core]
-version = "0.6.3"
+version = "0.6.2"
```

:wrench: Should this have gotten downgraded?
```suggestion
version = "0.6.3"
```

---

Done.